deepchem/models/torch_models/tests/test_progressive_multitask.py

```diff
@@ -14,30 +14,29 @@
 
 
 @pytest.mark.torch
-def test_construction():
-    """
-    Test that PrgressiveMultiTask Model can be constructed without crash.
+def test_progressivemultitask_construction():
+    """Test that PrgressiveMultiTask Model can be constructed without crash.
     """
 
     model = dc.models.torch_models.ProgressiveMultitask(
         n_tasks=1,
+        mode="classification",
         n_features=100,
         layer_sizes=[128, 256],
         alpha_init_stddevs=[0.08],
         weight_init_stddevs=0.02,
         bias_init_consts=1.0,
         activation_fns=nn.ReLU,
         dropouts=[0.2],
-        n_outputs=2,
```

Comment:
Is n_outputs in the TF model?

Reply:
Let's re-add this, and add a deprecation warning to this field in favor of n_classes. We shouldn't break backwards compatibility

Reply:
I think it will be better to add it as `n_classes` in this base model and instead keep it as `n_outputs` in the TorchModel wrapper with a deprecation warning. Else we will need to have two deprecation warnings, one for this and the other for the TorchModel class. Since it is the TorchModel that will be an equivalent model to the existing tf Model, we will be able to maintain backwards compatibility by keeping it as `n_outputs` there.

Reply:
Documenting discussion, we decided this should be in the TorchModel wrapper class and not in the pytorch module itself